docs/reference/commandline/build.md

```diff
@@ -80,6 +80,10 @@ the command succeeds, the directory is sent to the Docker daemon as the
 context. Local clones give you the ability to access private repositories using
 local user credentials, VPN's, and so forth.
 
+> **Note:**
```

Comment:
This change is unrelated to `docker attach` and should probably be submitted in a separate PR. I think this time we can let it go, but in future, please try to keep your PRs to one issue at a time, for ease of review and for if the change ever needs to be reverted.